variants/arduino_giga_r1_stm32h747xx_m7/variant.cpp

```diff
@@ -6,3 +6,12 @@ void _on_1200_bps() {
     *(__IO uint32_t *)tmp = (uint32_t)0xDF59;
     NVIC_SystemReset();
 }
+
+#if defined(CONFIG_BOARD_ARDUINO_GIGA_R1)  && defined(CONFIG_INPUT_GT911_INTERRUPT)
```

Comment:
Is this needed? A previous sketch or build means you reset the board so BSS variables get cleared. Am I missing something? Can it reload another sketch? If so, this needs to be somewhere else executed by all boars.

Reply:
Sorry, I don't know well enough how the ltext stuff works, when you ask it to load a new sketch.  Does it go
all the way through the "Real" bootloader and potentially make a clean load of the Zephyr loader code and
then do a clean install of the new sketch?  Or does it simply jump to the loader of the ltext and potentially leave
in things that may have changed values...  If the later is possible that is why I put that part in... 

If you prefer, I Can easily pull it out for now.  And if we run into some strange issue later... Address it then.

Reply:
Either way, the bootloader would need to jump back to the loader which would then clear BSS. I don't think it's needed, but you could test it:
```C
void zephyr_input_register_callback(zephyr_input_callback_t cb) {
        assert(!zephyr_input_cb);
	zephyr_input_cb = cb;
```
Assuming sketch loading works, which doesn't seem to work right now, so you need double tap anyway.